docs/learn/front-end/typescript/exercises/02-single-counter/counter.tsx

```diff
@@ -0,0 +1,60 @@
+import { component as component_ } from "front-end/lib/framework";
+import { adt, ADT } from "shared/lib/types";
+import React from "react";
+import { ComponentViewProps } from "front-end/lib/framework/component/base";
```

Comment:
Convention is to namespace framework imports, so this would be `component_.base.ComponentViewProps`. I arbitrarily chose this convention during the refactor, so I'm not personally attached to it. However, I find it keeps things consistent.

Reply:
I think it makes sense to keep it consistent 👍